src/functions/process-stream-event.ts

```diff
@@ -101,11 +104,12 @@ export const processStreamEvent: Handler = async (
 
         await convert(tableName, operationType, image);
```

Comment:
What happens if an error is thrown here and caught in the catch block, prior to printLogs() being called? Is there any information in the ILog that could still be useful, or are we happy there is enough information in the error logs in the catch block?

Reply:
The SQS message id gets logged on error and the debug logs can be switched on if there is an issue that needs investigating. Happy to get the general consensus on it though! 😸